manim/cli/render/render_options.py

```diff
@@ -106,6 +103,27 @@ def validate_resolution(ctx, param, value):
         type=click.Path(),
         help="The path to the WebGL frontend.",
     ),
+    option(
+        "-i",
```

Comment:
This was `-g` before (and having both `--save_pngs` as well as `--save_as_gif` shorthand to `-i` doesn't make much sense I think).

```suggestion
        "-g",
```

Reply:
Yes, I noticed that too. I haven't really used either of these flags much so that was a copy/paste error. I assumed that's because if you wanted to have save_pngs/save_as_gif you could do both at the same time. Tested this locally after submitted through GUI and this produces pngs/gifs still.